### ISSUE TITLE
Duplicated printout from cmsRun.cc

### ISSUE THREAD

**Issue by davidlange6** [cms-sw/cmssw#18757]
Hi @Dr15Jones - its interesting that this is printed twice. A sign of a problem? (eg, step3 wf 1325)

%MSG-i ThreadSetup:  (NoModuleName) 16-May-2017 11:23:45 CEST pre-events
setting # threads 8
%MSG
%MSG-i StreamSetup:  (NoModuleName) 16-May-2017 11:23:45 CEST pre-events
setting # streams 8
%MSG

**Comment by Dr15Jones** [comment]
@davidlange6 do you have an example of it printing twice? The example you gave is not a double print since one gives the number of streams and the other the number of threads.

**Comment by davidlange6** [comment]
Ok, I'm sleeping.. but let me rephrase, could we reduce this printout from 6 lines to 1 (or max 3)

**Comment by Dr15Jones** [comment]
Would
```
%MSG-i ThreadStreamSetup:  (NoModuleName) 16-May-2017 08:45:54 CDT pre-events
setting # threads 2
setting # streams 2
%MSG
```
be acceptable?

**Comment by Dr15Jones** [comment]
(Partially?) address in #18833

**Comment by Dr15Jones** [comment]
assign core

**Comment by Dr15Jones** [comment]
+1
